Adding FlappyFirebolt Demo

mohtashin opened the pull request:
Flappy Firebolt Edition

A lightweight Flappy Bird clone built in **Python + Pygame** that stores scores in a **Firebolt** database.

Commit:
Adding FlappyFirebolt Demo

fb-cole reviewed (commented):
We should have something in the demo that explains the point of it... is it that you could have an entire company of 10,000 people playing and writing scores? Should we generate fake data as part of the app so you can see a constantly-evolving scoreboard that updates in real time?

Commit:
Update FlappyFirebolt README based on PR feedback

- Fix clone URL to point to firebolt-demo repo
- Clarify .env file location as FlappyFirebolt folder
